node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -45,6 +46,36 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
 
   val peers: Vector[Peer]
 
+  val peerServices: mutable.Map[Peer, ServiceIdentifier] =
```

Comment:
This is a publicly available mutable variable. Can this be made `private`?

Reply:
Done